deploy/devcontainer-feature/src/radcli/README.md

```diff
@@ -8,7 +8,7 @@ This will install the latest stable release of the `rad` CLI.
 
 ```json
 "features": {
-    "ghcr.io/devcontainers/radius/radiuscli:latest": {
+    "ghcr.io/radius-project/devcontainer-features/radcli:latest": {
```

Comment:
This is what the updated path will be. `ghcr.io/radius-project/radius/radcli:latest` is the current path.